tests/Core/Ruleset/PopulateTokenListenersRegisterNoArrayTest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="PopulateTokenListenersTest" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/PHPCSStandards/PHP_CodeSniffer/master/phpcs.xsd">
```

Comment:
No, the way I've done it throughout these (new) tests is to have the name match the name of the test file, not of the test fixture. That way the fixture is directly linked to the test file in which it is being used (as there are sometimes multiple test files for a method with even more XML fixtures, so it can get confusing which fixture belongs with which test).

Also see #700.